### DISCUSSION TITLE
Does the project is still alive

### DISCUSSION THREAD

**Discussion by PPCM** [OctoFarm/OctoFarm#1396] [General]
Hello the team,

Last update was almost a year ago, does the project still alive, because, it looks great
Thanks for your help

**Comment by wicada** [comment]
No, all development has stopped on this project.

**Reply by wicada** [reply]
There was an announcement made to the Patreon supporters from the developer on April 11th.

**Reply by PPCM** [reply]
Bad news, the project seems to be good

Do you know an open-source alternative?
